### PR TITLE
add badges landing page

### PR DESCRIPTION
**Pull request by SouravInsights**
## PropLot Badges
This PR adds a new page for claiming Sismo Badges for all eligible PropLot contributors. For for context, please take a look at this [discussion](https://discord.com/channels/954142017556979752/1014583865513349230) on Discord & the [proposal](https://prop.house/proposal/1371) behind this idea.

**Figma Mockup:** https://www.figma.com/file/pHD4lxrfJAJ1mdUfFzCDVy/LilNouns-PropLot-Badges

**Note:** I had to modify the layout of the NavWallet Dropdown Menu to accommodate the new `badge` nav item. Below you can see how it looks like now:
CleanShot 2022-09-24 at 15 31 45@2x

For a quick review, please take a look at the below screenshots of the new page in different screen sizes:

#### Desktop:
CleanShot 2022-09-24 at 15 59 27@2x

#### Mobile:
CleanShot 2022-09-24 at 15 39 31@2x

### CODE REVIEW

File: packages/nouns-webapp/src/components/NavWallet/NavWallet.module.css [hunk 1]
```diff
@@ -165,13 +149,14 @@
   margin-top: 0.3rem;
 }
 
-.connectContentMobileText {
```
Comment (mcgingras) [inline packages/nouns-webapp/src/components/NavWallet/NavWallet.module.css:168]: is it okay that we are wiping out this class by renaming it? It's not being used anywhere else?

**Review by mcgingras — APPROVED** [review]
I like it, looks pretty straightforward and a good addition!

File: packages/nouns-webapp/src/components/NavWallet/NavWallet.module.css (hunk 1 above)
Comment (SouravInsights) [reply packages/nouns-webapp/src/components/NavWallet/NavWallet.module.css:168]: nope! it's not used anywhere else. regarding the className, I just thought `connectContentMobileRoutes` will make more sense.

**Comment by SouravInsights** [comment]
> I like it, looks pretty straightforward and a good addition!

Thanks! 🙏

**Pull request merged**